### ISSUE TITLE
Room List should act a navigation treeview widget

### ISSUE THREAD

**Issue by t3chguy** [vector-im/riot-web#11277]
Most of the work here is done but the entire widget should have just one tab stop, the selected room tile and use only arrow-keys for navigation.

https://www.w3.org/TR/wai-aria-practices/examples/treeview/treeview-2/treeview-2a.html

**Comment by t3chguy** [comment]
Closing in favour of https://github.com/vector-im/riot-web/issues/11205
